openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -61,6 +61,7 @@ class CourseInfoSerializer(serializers.Serializer):  # pylint: disable=abstract-
     Compare this with CourseDetailSerializer.
     """
 
+    can_show_upgrade_sock = serializers.BooleanField()
```

Comment:
I don't think we should change anything, but one thing that's interesting is that can_load_courseware returns a dict describing the reason a user may not be able to load it - like a boolean with extra metadata.  It doesn't quite make sense for this, but I wonder if we eventually want an API standard that says that permissions are expressed as `can_*` and are all described as AccessResponses.  Anyway, this looks good. :)